src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -51,40 +123,93 @@ Charge trapping model presented in [Boggs _et al._ (2023)](https://doi.org/10.10
 * `nσe::T`: Trapping product for electrons (default: `(nσe)^-1 = 1020cm`).
 * `nσh::T`: Trapping product for holes (default: `(nσh)^-1 = 2040cm`).
 * `temperature::T`: Temperature of the crystal (default: `78K`).
+* `inactive_layer_geometry`: The geometry of the inactive layer. If it's not defined, it uses point_types to get the inactive layer geometry, else it uses the user-defined inactive_layer_geometry.
```

Comment:
I'm a bit confused by this because `BoggsChargeTrappingModel` can also just be applied to the bulk (?)